lib/chainsync/chainsync/exec/acquire_data.py

```diff
@@ -77,24 +77,18 @@ def acquire_data(
     # Get last entry of pool info in db
     data_latest_block_number = get_latest_block_number_from_pool_info_table(db_session)
     # Using max of latest block in database or specified start block
-    block_number: BlockNumber = BlockNumber(max(start_block, data_latest_block_number))
-    # Make sure to not grab current block, as the current block is subject to change
-    # Current block is still being built
-    latest_mined_block = hyperdrive.get_block_number(hyperdrive.get_current_block())
-    lookback_block_limit = BlockNumber(lookback_block_limit)
-    if (latest_mined_block - block_number) > lookback_block_limit:
-        block_number = BlockNumber(latest_mined_block - lookback_block_limit)
+    curr_write_block = max(start_block, data_latest_block_number + 1)
+
+    latest_mined_block = int(hyperdrive.get_block_number(hyperdrive.get_current_block()))
```

Comment:
might as well rename `hyperdrive` to `interface` while you're in here to stay consistent with the rest of the codebase.